examples/binaries/new-meta/tests/read_state.rs

```diff
@@ -61,6 +65,32 @@ fn read_state_bytes_with_parameterized_wasm_func_returns_transformed_state() {
     assert_eq!(expected_state, actual_state);
 }
 
+#[test]
+fn read_state_bytes_with_two_args_wasm_func_returns_transformed_state() {
+    let system = System::new();
+    let program = initialize_current_program(&system);
+    const WALLET_BY_NAME_AND_SURNAME_FUNC_NAME: &str = "wallet_by_name_and_surname";
```

Comment:
Too long name for local const.
```suggestion
    const FUNC_NAME: &str = "wallet_by_name_and_surname";
```